### DISCUSSION TITLE
Detect-adaptive and thresholds

### DISCUSSION THREAD

**Discussion by Boulder08** [Breakthrough/PySceneDetect#357] [General]
I'm a little unsure which threshold is being respected with detect-adaptive. I have been testing the detection and I've noticed that it only picks ones where both adaptive_ratio and content_val is past the limit. There are some clear cuts where adaptive_ratio is less than the threshold like there in frame 168422.

Is it so that both need to be exceeded?

`1135,168322,01:57:00.388,7020.388,169274,01:57:40.136,7060.136,953,00:00:39.748,39.748
1136,169275,01:57:40.136,7060.136,169383,01:57:44.683,7064.683,109,00:00:04.546,4.546`

`168422,01:57:04.559,2.8025076826107735,29.109786883503403,129.76323341836735,15.873804209183673,31.051352572278912,40.40420386904762`

**Comment by Breakthrough** [comment]
Yes, both need to be exceeded.  The [adaptive detector](https://www.scenedetect.com/docs/0.6.2/cli.html#detect-adaptive) allows you to specify a threshold for [`adaptive_ratio` (`-t`)](https://www.scenedetect.com/docs/0.6.2/cli.html#cmdoption-scenedetect-detect-adaptive-t) as well as a threshold for [`content_val` (`-c`)](https://www.scenedetect.com/docs/0.6.2/cli.html#cmdoption-scenedetect-detect-adaptive-c).
